### PR TITLE
BaSP-MR-64: Created test for delete and put request

### PR DESCRIPTION
**Pull request by santimenendez98**
Created test for delete and put request, these tests check different things about requests.

### CODE REVIEW

File: src/tests/trainer.test.js [hunk 1]
```diff
@@ -72,3 +64,101 @@ describe('POST /api/trainer', () => {
...
+      city: 'Boston',
+      salary: 324,
+    };
+    const response = await request(app).put(`/api/trainer/${id}`).send(data);
+    expect(response.status).toBe(500);
+    expect(response.error).toBeTruthy();
+  });
+});
+
+describe('DELETE /api/trainer/:id', () => {
```
Comment (josedaniele) [inline src/tests/trainer.test.js:131]: the same as in the put

File: src/tests/trainer.test.js [hunk 2]
```diff
@@ -72,3 +64,101 @@ describe('POST /api/trainer', () => {
     expect(response.error).toBeTruthy();
   });
 });
+
+describe('PUT /api/trainer/:id', () => {
```
Comment (josedaniele) [inline src/tests/trainer.test.js:68]: where is the case of non-existent id?

**Review by josedaniele — APPROVED** [review]

**Review by danielninfi — APPROVED** [review]
pass all tests

File: src/tests/trainer.test.js [hunk 3]
```diff
@@ -7,14 +7,6 @@ beforeAll(async () => {
   await Trainer.collection.insertMany(trainerSeed);
 });
 
-describe('GET /api/trainer', () => {
```
Comment (JuanMornik) [inline src/tests/trainer.test.js:10]: why did you deleted this test?

**Review by JuanMornik — CHANGES_REQUESTED** [review]

File: src/tests/trainer.test.js [hunk 4]
```diff
@@ -72,3 +64,128 @@ describe('POST /api/trainer', () => {
...
+  });
+});
+
+describe('GET /api/trainer', () => {
+  test('Should return all elements', async () => {
+    const response = await request(app).get('/api/trainer').send();
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+  test('Should return if the database has no objects', async () => {
```
Comment (JuanMornik) [inline src/tests/trainer.test.js:185]: Missing status 500 test

File: src/tests/trainer.test.js (hunk 3 above)
Comment (JuanMornik) [reply src/tests/trainer.test.js:10]: I didn't see it, don't pay attention to this comment

**Review by JuanMornik — CHANGES_REQUESTED** [review]

**Review by JuanMornik — APPROVED** [review]

**Pull request merged**